pkgs/top-level/all-packages.nix

```diff
@@ -18329,6 +18329,8 @@ with pkgs;
     inherit (darwin) ditto;
   };
 
+  fennel-ls = callPackage ../development/tools/language-servers/fennel-ls { };
```

Comment:
We are currently in the process of moving packages to [`pkgs/by-name`](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name). Just recently, there was a PR which updates the checker so that you're not allowed to add new packages (with some exceptions) into non-by-name locations. Would you mind moving the package to `pkgs/by-name/fe/fennel-ls/package.nix`? You can remove this line afterwards.

Reply:
Thanks for all the suggestions, implemented all except for this last one. 3:18 am here.

Reply:
@yisraeldov Could you have another look at moving it to `pkgs/by-name`? It's kinda required now, as #275539 has been merged.

Reply:
@h7x4 Ok, I moved it to `pkgs/by-name`